### ISSUE TITLE
Science from Evangelism not triggering

### ISSUE THREAD

**Issue by bklick** [LoneGazebo/Community-Patch-DLL#3915]
_1. Mod version (i.e Date - 4/23):_

1/26
_2. Mod list (if using Vox Populi only, leave blank):_
All of VP
Historical Religions
Convenient Start
Meaningful City State Colors
(7) Custom events for VP
Cultural Diversity
Promotion Overhaul for VP

_3. Error description:_ 
Using a missionary to spread my civ's religion to a city with a different majority religion did not result in bonus science.
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1670609/Logs.zip)

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by UndeadDevel** [comment]
Reporting that this seems to still be an issue in 2/27; don't have any logs, though, sorry. Also my gamestate may be corrupted (long story; see forum thread of 2/27 version, post https://forums.civfanatics.com/threads/new-version-february-27th-2-27.629426/page-6#post-15060051)

**Comment by Edaka** [comment]
Yep, it's an issue in my game, too. What kind of supplement would be useful? A save?

**Comment by LoneGazebo** [comment]
Did you actually convert anyone?

**Comment by UndeadDevel** [comment]
I did. I converted citizen units in the following scenarios and watched my science:

1) no majority religion
2) majority religion without flipping the city
3) majority religion with flipping to none
4) majority religion with flipping to mine

In none of those cases did I get any extra science.

**Comment by LoneGazebo** [comment]
Can I get a save and zipped MODS folder?

**Comment by UndeadDevel** [comment]
Sure. [Here you go.](https://ufile.io/6iz1n)
I placed two missionaries near the Ottoman to the east of my kingdom. Using the one near Bursa will flip the city to my religion; using the one near Istanbul will not but will still convert some citizens. I flipped Bursa the turn before while testing so you won't see the science and production bonus from Council of Elders but Istanbul I didn't touch yet.
